src/app/component/dashboard/dashboard.component.ts

```diff
@@ -1,57 +1,25 @@
 import { Component,OnDestroy,OnInit } from '@angular/core';
 import { Params, Router } from '@angular/router';
 import quizData from '../../../assets/json/data.json';
-import { QuestionService } from '../../service/question.service';
-
 @Component({
   selector: 'app-dashboard',
   templateUrl: './dashboard.component.html',
   styleUrls: ['./dashboard.component.css']
 })
 export class DashboardComponent implements OnInit,OnDestroy {
-  quizData = quizData;
+  quizData = {...quizData};
   title: any;
   arrayIndex=0;
-  image: any;
-  description: any;
-  QuizType:any = [0,1,2];
+  QuizType:any;
```

Comment:
here too and please give name like quizType:string

Reply:
yes i have given type and also changed the name